src/app/components/shared/cards/card-image/card-image.component.ts

```diff
@@ -16,60 +16,24 @@ import { Card } from "../cards.component";
   selector: "baw-card-image",
   styleUrls: ["./card-image.component.scss"],
   template: `
-    <div class="card h-100">
-      <!-- Card Image -->
-      <ng-container *ngIf="card.link || card.route; else noLinkImage">
-        <ng-container *ngIf="card.link; else route">
-          <a [href]="card.link">
-            <img [alt]="card.title + ' image'" [src]="card.model.image" />
-          </a>
-        </ng-container>
-
-        <ng-template #route>
-          <a [bawUrl]="card.route">
-            <img [alt]="card.title + ' image'" [src]="card.model.image" />
-          </a>
-        </ng-template>
-      </ng-container>
-
-      <ng-template #noLinkImage>
-        <img [alt]="card.title + ' image'" [src]="card.model.image" />
-      </ng-template>
-
-      <!-- Card Body -->
-      <div class="card-body">
-        <!-- Title -->
-        <h4 class="card-title">
-          <ng-container *ngIf="card.link || card.route; else noLinkTitle">
-            <!-- External header link wrapper -->
-            <ng-container *ngIf="card.link; else route">
-              <a [href]="card.link">{{ card.title }}</a>
-            </ng-container>
-
-            <!-- Internal header link wrapper -->
-            <ng-template #route>
-              <a [bawUrl]="card.route">{{ card.title }}</a>
-            </ng-template>
-          </ng-container>
-
-          <!-- Header -->
-          <ng-template #noLinkTitle>{{ card.title }}</ng-template>
-        </h4>
-
-        <!-- Card Description -->
-        <!-- Line truncation fails in SSR https://github.com/DiZhou92/ngx-line-truncation/issues/49 -->
-        <p
-          *ngIf="!isServer; else noTruncation"
-          class="card-text"
-          [ngClass]="{ 'font-italic': !card.description }"
-          [line-truncation]="4"
-          [innerHTML]="description"
-        ></p>
-        <ng-template #noTruncation>
-          <p>Loading</p>
-        </ng-template>
+    <a class="text-reset" [bawUrl]="card.route">
+      <div class="card h-100">
+        <!-- Image -->
+        <div class="card-image">
+          <img [alt]="card.title + ' image'" [src]="card.model.image" />
+        </div>
+
+        <div class="card-body">
+          <!-- Title -->
+          <h4 class="card-title" [innerText]="card.title"></h4>
+
+          <!-- Description -->
+          <div class="card-text truncate">
+            <p [innerHtml]="description"></p>
+          </div>
+        </div>
```

Comment:
i'd say the entire card being clickable could be annoying. IMO just the image and title should be clickable